src/Scaffolding/VS.Web.CG.EFCore/DbContextEditorServices.cs

```diff
@@ -161,39 +169,25 @@ public EditSyntaxTreeResult EditStartupForNewContext(ModelType startUp, string d
 
                 var configServicesMethod = startUpClassNode.ChildNodes()
                     .FirstOrDefault(n => n is MethodDeclarationSyntax
-                        && ((MethodDeclarationSyntax)n).Identifier.ToString() == "ConfigureServices") as MethodDeclarationSyntax;
-
+                        && ((MethodDeclarationSyntax)n).Identifier.ToString() == ConfigureServices) as MethodDeclarationSyntax;
                 var configRootProperty = TryGetIConfigurationRootProperty(startUp.TypeSymbol);
-
+                //if using Startup.cs, the ConfigureServices method should exist. 
                 if (configServicesMethod != null && configRootProperty != null)
                 {
                     var servicesParam = configServicesMethod.ParameterList.Parameters
-                        .FirstOrDefault(p => p.Type.ToString() == "IServiceCollection") as ParameterSyntax;
-                    string textToAddAtEnd;
+                        .FirstOrDefault(p => p.Type.ToString().Equals(IServiceCollection));
+                        
                     var statementLeadingTrivia = configServicesMethod.Body.OpenBraceToken.LeadingTrivia.ToString() + "    ";
+                    string textToAddAtEnd = AddDbContextString(minimalHostingTemplate: false, useSqlite, statementLeadingTrivia);
```

Comment:
This can move inside the if block below, no?

Reply:
yep will move it.